### ISSUE TITLE
[Suggestion] Allow return to save account name

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#18050]
## Description 

Allow return to save account name

## Steps to Reproduce

   1. Create a new profile on nightly and enable native wallet
   2. Update account name 
   3. Return to save name, doesn't work

## Actual result:

Return key doesn't save name

## Expected result:
Allow return to save account name

## Reproduces how often: 

Easy

## Brave version (brave://version info)

Brave | 1.31.33 Chromium: 93.0.4577.63 (Official Build) nightly (x86_64)
-- | --
Revision | `ff5c0da2ec0adeaed5550e6c7e98417dac77d98a-refs/branch-heads/4577@{#1135}`
OS | All

## Version/Channel Information:

- Can you reproduce this issue with the current release? NA
- Can you reproduce this issue with the beta channel? NA
- Can you reproduce this issue with the nightly channel? Yes

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

## Miscellaneous Information:

cc: @bbondy @Douglashdaniel this is to support keyboard accessibility

**Comment by bbondy** [comment]
When editing account names?

**Comment by srirambv** [comment]
Yes while editing account names.
